### PR TITLE
use const and let instead of var

### PR DESCRIPTION
**Pull request by stephenwade**
Since we're using ES2015, we have `let` and `const`, which are better than `var` for declaring variables.

### CODE REVIEW

**Comment by stephenwade** [comment]
I signed it!

File: step-09/my-carousel.html [hunk 1]
```diff
@@ -173,18 +173,18 @@
...
         }
 
         if (this._touchDir === 'x') {
           // Prevent vertically scrolling when swiping
           event.preventDefault();
 
-          var dx = Math.round(event.changedTouches[0].clientX - this._startX);
-          var prevChild = this.selected.previousElementSibling;
-          var nextChild = this.selected.nextElementSibling;
+          const dx = Math.round(event.changedTouches[0].clientX - this._startX);
```
Comment (keanulee) [inline step-09/my-carousel.html]: Use `let` -`dx` may be modified below (L191)

File: step-09/my-carousel.html [hunk 2]
```diff
@@ -213,17 +213,17 @@
         }
 
         if (this._touchDir === 'x') {
-          var dx = Math.round(event.changedTouches[0].clientX - this._startX);
-          var prevChild = this.selected.previousElementSibling;
-          var nextChild = this.selected.nextElementSibling;
+          const dx = Math.round(event.changedTouches[0].clientX - this._startX);
```
Comment (keanulee) [inline step-09/my-carousel.html]: Use `let` - `dx` may be modified below (L222)

**Review by keanulee — CHANGES_REQUESTED** [review]
`const` can also be used for `MyMixin` in my-mixin.html

This is great, thanks @stephenwade!

File: step-09/my-carousel.html (hunk 2 above)
Comment (stephenwade) [reply step-09/my-carousel.html]: Fixed in b29d661.

**Comment by stephenwade** [comment]
`MyMixin` now uses `const`, and `dx` uses `let`.

**Review by keanulee — APPROVED** [review]

**Comment by keanulee** [comment]
Codelab will be updated with these changes shortly

**Pull request merged**